src/framework/control.ml

```diff
@@ -470,15 +470,16 @@ struct
           if GobConfig.get_bool "incremental.save" then
             Serialize.store_data solver_data Serialize.SolverData;
           if save_run <> "" then (
-            let analyses = Filename.concat save_run "analyses.marshalled" in
-            let config = Filename.concat save_run "config.json" in
-            let meta = Filename.concat save_run "meta.json" in
-            let solver_stats = Filename.concat save_run "solver_stats.csv" in (* see Generic.SolverStats... *)
-            let cil = Filename.concat save_run "cil.marshalled" in
-            let warnings = Filename.concat save_run "warnings.marshalled" in
-            let stats = Filename.concat save_run "stats.marshalled" in
+            let save_run = Fpath.v save_run in
```

Comment:
Maybe not name it `save_run` here, it is a bit confusing as the string `save_run` is used again later, which one might get confused if one doesn't see this let is inside an if.